src/components/Welcome.js

```diff
@@ -1,9 +1,10 @@
 import React from 'react';
 
-const Welcome = () => {
+const Welcome = (props) => {
```

Comment:
Nit-pick: Adding propTypes are a great way to warn if forgetting to add pass a prop to a component or if passing the wrong data type as a prop. They're also a great way to provide documentation to your code. https://reactjs.org/docs/typechecking-with-proptypes.html